library/std/src/sys/unix/mod.rs

```diff
@@ -195,13 +195,41 @@ pub fn cvt_nz(error: libc::c_int) -> crate::io::Result<()> {
     if error == 0 { Ok(()) } else { Err(crate::io::Error::from_raw_os_error(error)) }
 }
 
-// On Unix-like platforms, libc::abort will unregister signal handlers
-// including the SIGABRT handler, preventing the abort from being blocked, and
-// fclose streams, with the side effect of flushing them so libc buffered
-// output will be printed.  Additionally the shell will generally print a more
-// understandable error message like "Abort trap" rather than "Illegal
-// instruction" that intrinsics::abort would cause, as intrinsics::abort is
-// implemented as an illegal instruction.
+// libc::abort() will run the SIGABRT handler.  That's fine because anyone who
```

Comment:
Missing code-quotes in a bunch of places. Also for some of these you may be able to use intra-doc links like:

    [`foo`]

Reply:
Note that this isn't a doc comment but a normal comment, so adding `[]` doesn't make sense.